tunup/check_tunnel_status.py

```diff
@@ -0,0 +1,33 @@
+import logging
+from time import sleep, gmtime
+import requests
+import sys
+
+t = int(sys.argv[1])
```

Comment:
sys was good for when we were building it but now that were actually merging it this should be `argparse`

```py
import argparse

parser = argparse.ArgumentParser()
parser.add_argument(
  "--request_interval_seconds",
  type=int,
  default=5,
  help="Interval to query flightradar24"
)
parser.add_argument(
  "--hosts",
  type=str,
  help="comma separated values for hosts to send requests to \
    i.e. http://host.docker.internal:11000/api/health-check,http://host.docker.internal:14000/healthcheck/printer"
)

args = parser.parse_args()

print(args.request_interval_seconds)
print(args.hosts)

hosts_list = [host.strip for host in args.hosts.split(',')]

# iterate through hosts list and u can ping it easy
```